settings/settings.go

```diff
@@ -182,6 +182,19 @@ func (conf *Settings) NewIntegerValue(service, name string, value int64) error {
 	return conf.NewValue(service, name, TypeInteger, value)
 }
 
+// VerifyType to make sure type is valid
+func (conf *Settings) VerifyType(sType string) bool {
+	validTypes := map[string]bool{
+		TypeString:  true,
+		TypeBoolean: true,
+		TypeInteger: true,
+	}
+	if !validTypes[sType] {
+		return false
+	}
+	return true
+}
```

Comment:
if you want to do this with a map, i'd make `validTypes` be a global so you're not reconstructing it every time this is called. then you'd check it like

```
var validTypes = map[string]struct{}{
  TypeString: struct{}{},
  TypeBoolean: struct{}{},
  TypeInteger: struct{}{},
}

func (conf *Settings) VerifyType(sType string) bool {
  _, ok := validTypes[sType]
  return ok
}
```